db/data/20221011140515_add_missing_disabilities_from_tad_check.rb

```diff
@@ -72,6 +76,8 @@ def other_disability
   end
 
   def csv
-    @csv ||= CSV.table(Rails.root.join("data/tad_disability_check_2022-10-03.csv"))
+    return unless Rails.root.join(FILE).exist?
+
+    @csv ||= CSV.table(Rails.root.join(FILE))
```

Comment:
This is unrelated to the context of this PR, but removes a redundant CSV (see guidance for review)